packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -611,6 +615,11 @@ $navsidebar-sub-bg-active: $navsidebar-sub-bg;
 
 $navsidebar-border-color: $navsidebar-bg-hover;
 
+//== Form
+//## Used in components/inputs
+
+// Form Label
+$form-label-color: $brand-inverse !default;
```

Comment:
I don't think this is meant to be `default!`, right?

Reply:
Correct. Removed the `!default` flag.